src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import { Home, Repos } from './pages';
+
+const App = () => {
+
+    return (
+        <Switch>
+            <Route path = '/:username'>
+                <Repos/>
+            </Route>
+            <Route path = '/'>
```

Comment:
Feel like you potentially might need an exact in here?

Reply:
our plan was to send them back to the for in case of error